emission/tests/netTests/TestStats.py

```diff
@@ -6,10 +6,88 @@
 
 # Our imports
 from emission.core.get_database import get_client_stats_db, get_server_stats_db, get_result_stats_db
+from emission.net.int_service.giles import archiver
 from emission.net.api import stats
 
 logging.basicConfig(level=logging.DEBUG)
 
+class TestArchiver(unittest.TestCase):
```

Comment:
Please put this into its own class. It's not a simple two-liner, it's a substantial piece of code, so it makes sense to pull it out rather than sticking it into a file that doesn't even have the same name.